LICENSE.md

```diff
@@ -0,0 +1,9 @@
+The MIT License (MIT)
+
+Copyright (c) 2017 GitTut
```

Comment:
@debugpoint136 is the organization name okay?

Reply:
Yup looks good. 👍